Makefile

```diff
@@ -7,7 +7,7 @@ images:
 	./hack/images.sh $(DOCKER_REPO_OVERRIDE)
 
 install:
-	./hack/install.sh
+	./hack/install.sh $(PREVIOUS_SERVERLESS_VERSION)
```

Comment:
I think I'd prefer a `make install-previous` to not have to remember the incantation 😂